### ISSUE TITLE
Odd behavior when switching between forward and reverse on a double clutching transmission

### ISSUE THREAD

**Issue by Dudejo** [Mogli12/GearboxAddon#404]
Basically, I can switch between forward and reverse as if it was a tractor's Power Shuttle.

I am not required to lift the clutch, stop the truck or even go into Neutal.

I used a "Band-Aid" fix to rectify the issue but it's odd how the reverse gear seems to otherwise ignore the general requirements of shifting.

			<gears axleSpeed="3.73" tireWidth="315" tireRatio="80" rimDiameter="22.5" shiftTimeMs="500" reverseReset="false" doubleClutch="true">
				<gear name="L" maxRange="1" forwardOnly="true" inverseRatio="2.78"/>
				<gear name="R" reverseOnly="true" inverseRatio="2.99"/>
				<gear name="1" forwardOnly="true" inverseRatio="1.95"/>
				<gear name="2" forwardOnly="true" inverseRatio="1.38"/>
				<gear name="3" forwardOnly="true" inverseRatio="1.00"/>
				<gear name="4" forwardOnly="true" inverseRatio="0.73"/>
			</gears>
			
			<ranges doubleClutch="true" defaultRange="1">
				<range name="L" inverseRatio="3.78"/>
				<range name="H" inverseRatio="1.00"/>
			</ranges>

			<reverse onlyStopped="true"/>

**Comment by Dudejo** [comment]
ManualClutch will force a declutching requirement, yes. It will still let me engage reverse while moving, however.

autoStartStop has no effect on any requirements. However, I prefer it on "true" for double clutching purposes as it allows me to float the gears.

**Comment by Mogli12** [comment]
Ok, if you just want to make sure that you can change direction if you are fully stoppend then onlyStopped="true" is the right flag.

Technically, the gearbox does not engage the reverse gear while moving forwards. It will brake automatically.

**Comment by Dudejo** [comment]
yeah, combining onlyStopped and manualClutch will replicate a typical behavior or cars and trucks.

speaking of onlyStopped, this lock-out is common on mechanical range selectors for tractors, 4WD pick-ups and reverse gears for anything without a synchronized or power shuttle.
